### DISCUSSION TITLE
Set idle_timeout in Docker Container

### DISCUSSION THREAD

**Discussion by Yemtex** [drakkan/sftpgo#1463] [Q&A] [answered]
Hi,

I am unfortunately too incompetent to understand how to set the idle_timeout configuration in a Docker container.

I have read the "configuration" part of the docker readme and the "environment variables" part of the full-configuration readme. But I could not understand how to compose the environment variable for "idle_timeout".
Would it be "SFTPGO_COMMON__IDLE_TIMEOUT=0"?

Thanks you in advance.

**Comment by Yemtex** [comment]
How would somebody check the current running configuration?

**Comment by drakkan** [comment, marked as answer]
> Would it be "SFTPGO_COMMON__IDLE_TIMEOUT=0"?

That's correct

> How would somebody check the current running configuration?

please check the logs, the loaded configuration is logged at startup
